src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -54,7 +54,7 @@ def acr_build_show_logs(cmd,
     log_file_sas = build_log_result.log_link
 
     if not log_file_sas:
```

Comment:
Can RP ever return an empty `log_link` but the request itself succeeded?

Reply:
Basically I wanted to avoid this kind of validation on the client side. If the request succeeded, we should be able to safely assume RP returned a valid result.